source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/ReportDashboard.cshtml

```diff
@@ -16,50 +16,69 @@
+      <div class="content-selector-label">
+        <p>Período<p>
+      </div>
+      <div class="content-selector-dropdown">
+        @Html.DropDownList("selectedPeriod", (IEnumerable<SelectListItem>)ViewBag.ReportPeriods)
+      </div>
     </div>
-    <table class="table">
-        <thead>
-            <tr data-bind="foreach: headers">
-                <!-- ko if: sortable -->
-                <th data-bind="attr: { class: 'col-' + ($index()+1) }, click: $parent.sort, css: { selected: activeSort}">
-                    <i class="icon arrow"></i>
-                    <span data-bind="text: title"></span>
-                </th>
-                <!-- /ko -->
-                <!-- ko if: !sortable -->
-                <th data-bind="attr: { class: 'col-' + ($index()+1) }"><span data-bind="text: title"></span></th>
-                <!-- /ko -->
-            </tr>
-        </thead>
-        <tbody data-bind="foreach: items">
-            <tr>
-                <td data-bind="text: period"></td>
-                <td data-bind="text: fullName"></td>
-                <td data-bind="text: currentPosition"></td>
-                <td data-bind="text: seniority"></td>
-                <td data-bind="text: responsibleId"></td>
-                <td><span class="state-doc" data-bind="text: stateName, attr: { 'class': stateClasses }"></span></td>
-                <td><a class="link-responsable top-tip" data-bind="if: state() == 6, attr: {href: calificationUrl}" data-tips="Ver Evaluación"><i class="icon view"></i><span>Ver Evaluación</span></a></td>
-            </tr>
-        </tbody>
-    </table>
-    <script type="text/javascript">
-        var employeePeriod = "@ViewBag.Period";
-    </script>
+  </div>
+  <table class="table">
+    <thead>
+      <tr data-bind="foreach: headers">
+        <!-- ko if: sortable -->
+        <th data-bind="attr: { class: 'col-' + ($index()+1) }, click: $parent.sort, css: { selected: activeSort}">
+          <i class="icon arrow"></i>
+          <span data-bind="text: title"></span>
+        </th>
+        <!-- /ko -->
+        <!-- ko if: !sortable -->
+        <th data-bind="attr: { class: 'col-' + ($index()+1) }"><span data-bind="text: title"></span></th>
+        <!-- /ko -->
+      </tr>
+    </thead>
+    <tbody data-bind="foreach: items">
+      <tr>
+        <td data-bind="text: period"></td>
+        <td data-bind="text: fullName"></td>
+        <td data-bind="text: currentPosition"></td>
+        <td data-bind="text: seniority"></td>
+        <td class="responsible-column">
+          <span data-bind="if: isEvaluationManager">
+            <a class="calification-links edit top-tip" data-bind="click: showResponsibleManager">
+              <i class="icon edit"></i>
+              <span data-bind="text: responsibleId()"></span>
+            </a>
+          </span>
+          <!-- ko ifnot: isEvaluationManager -->
+          </span><span data-bind="text: responsibleId"></span>
+          <!-- /ko -->
```

Comment:
It is very minor, a matter of taste, but I prefer to have some symmetry in the code (at least when it is for free), I think that it makes the code easy to read and less error prone on changes.

The first _if_ is in a _element data binding_ (`<span data-bind="if: isEvaluationManager">`), and the _else_ in a _comment data binding_ (`<!-- ko ifnot: isEvaluationManager -->`), I think that it could be better to give both of them the same format.

Also, you are adding a new element just after close the previous one, again, it is a matter of taste, but I do not see it natural and in my opinion it looks confusing: `</span><span data-bind="text: responsibleId"></span>`.